external/source/exploits/installerfiletakeover/InstallerFileTakeOver/ReflectiveLoader.h

```diff
@@ -0,0 +1,215 @@
+//===============================================================================================//

```

Comment:
This already available at `external/source/ReflectiveDLLInjection/dll/src/ReflectiveLoader.h`.